monitoring/bentoml-plugins-arize/src/bentoml_plugins/arize/__init__.py

```diff
@@ -296,9 +296,9 @@ def __init__(
         model_version: str | None = None,
         environment: Environments | None = None,
         model_tags: dict[str, str | bool | float | int] | None = None,
-        **_: t.Any,
+        **kwargs: t.Any,
     ):
-        self.name = name
+        super().__init__(name, **kwargs)
```

Comment:
This returns a NotImplementedError here.

Reply:
@aarnphm That‘s because it requires the new sdk

Reply:
https://github.com/bentoml/BentoML/pull/3189

Reply:
I just workaround it by installing the main branch of bentoml